orchestra/static/orchestra/common/components/quill/quill.directive.es6.js

```diff
@@ -0,0 +1,239 @@
+import template from './quill.html'
+
+// TODO(jrbotros): update to v1.0.0
+import Quill from 'quill'
+import 'quill/dist/quill.base.css'
```

Comment:
This is Quill's internal css, but I accidentally also deleted our additional custom styles 😳 Added back in!